.changeset/deep-mangos-stand.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/datastore": minor
+"@fluid-private/test-end-to-end-tests": minor
+---
+---
+"section": feature
+---
```

Comment:
```suggestion
---
"@fluidframework/container-runtime": minor
"@fluidframework/datastore": minor
"__section": feature
---
```

The changeset format recently changed, but since this was created before the biold-tools update, we can just fix it manually.

Reply:
Done